features/vesting/hooks.ts

```diff
@@ -141,12 +140,11 @@ export const useVestingsLocked = (escrows: string[] | undefined) => {
       return BigNumber.from(0);
     }
     try {
-      return escrows
+      return await escrows
         .map((escrow) => vestingEscrowContractFactory(escrow, providerRpc))
         .map((contract) => contract.locked())
         .reduce(async (acc, cur) => {
-          const accValue = await acc;
-          const curValue = await cur;
+          const [accValue, curValue] = await Promise.all([acc, cur]);
           return accValue.add(curValue);
         }, Promise.resolve(BigNumber.from(0)));
```

Comment:
did not figure out first time, but why not to just do like this? (also goes to above call on 116)
```typescript
const escrowValues = await Promise.all(escrows...)
return escrowValues.reduce((a,c) => a.add(c))
```

Reply:
🔥